tutorials/Binary_Contingency_Scores.ipynb

```diff
@@ -0,0 +1,2738 @@
+       "  stroke: currentColor;\n",
+       "  fill: currentColor;\n",
+       "}\n",
+       "</style><pre class='xr-text-repr-fallback'>&lt;xarray.DataArray ()&gt; Size: 8B\n",
+       "array(0.83333333)</pre><div class='xr-wrap' style='display:none'><div class='xr-header'><div class='xr-obj-type'>xarray.DataArray</div><div class='xr-array-name'></div></div><ul class='xr-sections'><li class='xr-section-item'><div class='xr-array-wrap'><input id='section-62ffe7b9-2198-4c6f-845b-f12f84f0e41e' class='xr-array-in' type='checkbox' checked><label for='section-62ffe7b9-2198-4c6f-845b-f12f84f0e41e' title='Show/hide data repr'><svg class='icon xr-icon-database'><use xlink:href='#icon-database'></use></svg></label><div class='xr-array-preview xr-preview'><span>0.8333</span></div><div class='xr-array-data'><pre>array(0.83333333)</pre></div></div></li><li class='xr-section-item'><input id='section-6324a5e9-efeb-4444-8e57-6592e236525e' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-6324a5e9-efeb-4444-8e57-6592e236525e' class='xr-section-summary'  title='Expand/collapse section'>Coordinates: <span>(0)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><ul class='xr-var-list'></ul></div></li><li class='xr-section-item'><input id='section-66a9df28-cc72-4bcf-a65b-0d396aabc8c6' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-66a9df28-cc72-4bcf-a65b-0d396aabc8c6' class='xr-section-summary'  title='Expand/collapse section'>Indexes: <span>(0)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><ul class='xr-var-list'></ul></div></li><li class='xr-section-item'><input id='section-e1ac0d1f-46f5-4af6-b84c-f0b666f9c510' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-e1ac0d1f-46f5-4af6-b84c-f0b666f9c510' class='xr-section-summary'  title='Expand/collapse section'>Attributes: <span>(0)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><dl class='xr-attrs'></dl></div></li></ul></div></div>"
+      ],
+      "text/plain": [
+       "<xarray.DataArray ()> Size: 8B\n",
+       "array(0.83333333)"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "computed = contingency_manager.transform()\n",
+    "computed.accuracy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4a031199-cbc6-44fc-a906-dde83c0beca6",
+   "metadata": {},
+   "source": [
+    "## Exploring Dimensionality\n",
+    "\n",
+    "The transform function can also be utilised to explore the dimensionality of the data. This function accepts preserve_dims and reduce_dims as per the rest of the `scores` codebase. \n",
+    "\n",
+    "While the team behind `scores` intend to add functionality so that `scores` can support weights during transformation, this functionality has not yet been added.\n",
```

Comment:
We need an issue for this

Reply:
I have created an issue so it doesn't get lost. I also have an idea for how I think it should be implemented, but we'll deal with the options later. Thanks.